aws_acm_policies/aws_acm_certificate_expiration.py

```diff
@@ -5,6 +5,8 @@
 
 
 def policy(resource):
+    if not resource.get('NotAfter'):
```

Comment:
I made the assumption that if the NotAfter field in a cert is null, is it technically not expired. Let me know if that is incorrect. 
Also, would it be useful to create a policy that looks explicitly for null NotAfter fields, or … NotAfter fields that are way in the future? (I assume ACM wouldn’t allow “non-expiring” certs, but since it looks like people can technically upload their own)

Reply:
That sounds reasonable, perhaps there could be a separate policy at some point that alerts if `NotAfter` is not set, a `Low` severity policy maybe something like "ACM Certificate Does Not Expire" or something.